src/selectfield/selectfield.scss

```diff
@@ -50,17 +50,32 @@ $selectfield-error-color: unquote("rgb(222, 50, 38)") !default;
   border-radius: 0;
   -webkit-appearance: none;
   -moz-appearance: none;
+  -ms-appearance: none;
   appearance: none;
 
+  &:-moz-focusring {
```

Comment:
Firefox puts a dotted line around the select control when it gains focus by default. This eliminates that. I found it here [https://developer.mozilla.org/en-US/docs/Web/CSS/:-moz-focusring](url)